### ISSUE TITLE
Object reference not set to an instance of an object when selecting Generate mapping code

### ISSUE THREAD

**Issue by TomLac** [cezarypiatek/MappingGeneratorReleases#1]
Have always worked fine. But I get this error now always. restarted VS2019 and reinstalled last version of extension. But nothing is resolving the issue

System.NullReferenceException : Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.CSharp.CSharpExtensions.Kind(SyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetDeclaredSymbolCore(SyntaxNode node,CancellationToken cancellationToken)
   at MappingGenerator.Mappings.MappingLocationContext.FromContextExpression(SyntaxNode contextExpression,SemanticModel semanticModel)
   at MappingGenerator.Mappings.MappingContext..ctor(SyntaxNode contextExpression,SemanticModel semanticModel,MappingOptions options,SyntaxGenerator syntaxGenerator,MappingConfigPickupContext mappingConfigPickupContext)
   at async MappingGenerator.Features.Refactorings.Mappings.MappingMethod.MappingGeneratorRefactoring.GenerateMappingMethodBodyCustom(<Unknown Parameters>)
   at async MappingGenerator.Features.Refactorings.Mappings.MappingMethod.MappingGeneratorRefactoring.<>c__DisplayClass13_0.<TryToRegisterRefactoring>b__0(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetChangedSolutionAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.ComputeOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetPreviewOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.GetPreviewResultAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.<>c__DisplayClass11_0.<GetPreviewAsync>b__0(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformFunctionAsync[T](<Unknown Parameters>)

**Comment by cezarypiatek** [comment]
Hi @TomLac ,

Sorry for the late response. Can you provide a code where you try to execute MappingGenerator action? The method is a class member or it's a top-level statement?

**Comment by TomLac** [comment]
Hello ,

I have found what was causing this error.

I had declared a function that was accidently placed outside the curly brackets scope of the class.
When I tried to generate the default mappings, just by putting my mice over this option would cause the extensions to crash.
It was in the version for VS 2022.

So to reproduce this just add a class to your VS project create a function outside of your class and try to generate the default mappings.

Kindly regards

Tom Lacroix
